src/System.Management.Automation/engine/remoting/server/ServerRemoteHostUserInterface.cs

```diff
@@ -197,6 +197,14 @@ public override void WriteWarningLine(string message)
             _serverMethodExecutor.ExecuteVoidMethod(RemoteHostMethodId.WriteWarningLine, new object[] { message });
         }
 
+        /// <summary>
+        /// Read line as string masked.
+        /// </summary>
+        public override string ReadLineMaskedAsString()
+        {
+            throw new PSNotImplementedException();
```

Comment:
I would like some feedback here.
I think it makes sense to implement it for remote console as well and I started working on that but I realized that the `enum RemoteHostMethodId` has actually never been modified  and this tingled my spidey sense. 

Am I thinking something wrong here?

Reply:
Perhaps @PaulHigin could comment.

It is not clear for me about backward compatibility and the need to exрфтсу PSRP.

Reply:
Support for remoting would involve adding new host callbacks and this would modify the existing remoting protocol and may have implications for Windows PowerShell.  For now I think throwing this unsupported exception is fine.  But please create a new Issue for this (Remoting support for new ReadLineMaskedAsString host method).

Reply:
Thanks for the explanation, this makes sense. I created an issue https://github.com/PowerShell/PowerShell/issues/10935